src/registrar/templates/includes/required_fields.html

```diff
@@ -1,3 +1,3 @@
   <p class="margin-top-3">
-    Required fields are marked with an asterisk (<abbr class="usa-hint usa-hint--required" title="required">*</abbr>).
+    <i>Required fields are marked with an asterisk (<abbr class="usa-hint usa-hint--required" title="required">*</abbr>).</i>
```

Comment:
Haven't started an actual review yet, so this is a free-floating flag: Suggest using `<em>` instead of `<i>`. It makes sense semantically that the required note is emphasized.